README.md

```diff
@@ -50,3 +50,22 @@ Options:
   --[no]experimental: Calculate metrics for experimental runs.
     (default: false)
 ```
+
+### Generating end-of-year interop scores
+
+The scores used on the current year of the Interop Dashboard are generated
+regularly from this repository. At the end of each year, the scoring script
+should be run to generate final CSV files to use as the static final numbers
+for that interop year. Both stable and experimental numbers should be generated.
+
+```
+$ node interop-scoring/main.js --year={{YEAR}} --to={{YEAR + 1}}-01-02
```

Comment:
The rationale for choosing Jan 2nd here as the stopping point is that I assumed the final numbers would be the first run that appears on Jan 1st. No Jan 2nd runs should be processed because this denotes the stopping point for processing runs (exclusive).

The year parameter is more of a guideline on which interop categories to use and what date to start processing, and doesn't intrinsically add an end date explicitly. This can be changed, however, if we want to assume the end date if we don't have a `--to` flag provided.

Reply:
> I would say that we should rather use the last run before UTC midnight between Dec 31 and Jan 1, so that if we hover over the last point in the graphs, it says 2023-12-31 or earlier if there are no runs on that day.

I think that is a fine choice 🙂
I've updated the README, and I've updated the PR in wpt.fyi to reflect this in the data as well.

> can you double check what the last data point is now?

This would just change that the last data point is Jan 1 or Dec 31. Most interop results have Dec 31st aligned runs. Interop 2022 has the exception of having a last stable run on Dec 30th, but changing the date on the `--to` flag to Jan 2 does not intrinsically fix this problem, as sometimes there will be missing run days and there's not much we can do about this.